aws/naming_resources.md

```diff
@@ -0,0 +1,10 @@
+# Naming Resources
+
+## IAM Resources
+
+AWS operator needs a set of roles in customer accounts to be able to operator against the AWS API. Most of the customers need bounded policy to be security complaint. Here we define a convention to help them to achieve that and a same time have a coherent way to name the roles. From now own all clusters roles we need to create should have the prefix `gs-`. Example:
```

Comment:
Let's please wrap paragraphs by some amount of characters. 80 or 120 works well.

Reply:
done